mapie/conformity_scores/sets/aps.py

```diff
@@ -0,0 +1,147 @@
+from typing import Optional, cast
+
+import numpy as np
+from sklearn.dummy import check_random_state
+
+from mapie.conformity_scores.sets.naive import Naive
+from mapie.conformity_scores.sets.utils import get_true_label_cumsum_proba
```

Comment:
can we add `get_true_label_cumsum_proba` to the class method ? because it's only used for APS and RAPS. So if it's class method of APS, as RAPS inherits from APS, it will also be a method of RAPS

Reply:
Done 👍